src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import CalculatorDom from './components/calculator'; // eslint-disable line
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly note that all components should be class-based. In keeping with that kindly refactor this one to a class component.